app/models/exercise.rb

```diff
@@ -108,6 +110,10 @@ def icon_url
     "#{Exercism.config.website_icons_host}/exercises/#{icon_name}.svg"
   end
 
+  def mentoring_notes_url
```

Comment:
Nice refactor :)

```suggestion
  def edit_mentoring_notes_url
```

(as this takes you directly into GH's edit interface)

Reply:
Thanks!